# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/mysqlcc/qmakespec.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./configure.in.orig	2003-01-26 19:28:40.000000000 +0100
+++ ./configure.in	2003-01-26 19:28:49.000000000 +0100
@@ -25,7 +25,7 @@
 AC_SUBST(MYSQLCC_SOURCES)
 AC_OUTPUT(mysqlcc.pro)
 if test -z "${QMAKESPEC}"; then
-  QMAKESPEC=$QTDIR/mkspecs/$host_os
+  QMAKESPEC=$QTDIR/mkspecs/linux-g++
 else
   QMAKESPEC=${QMAKESPEC}
 fi
